released/packages/coq-ollibs/coq-ollibs.2.0.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "OL libraries"
+description: """
+  Add-ons for the Coq standard library
+"""
+
+homepage: "https://github.com/olaure01/ollibs"
+dev-repo: "git+https://github.com/olaure01/ollibs.git"
+bug-reports: "https://github.com/olaure01/ollibs/issues"
+doc: "https://github.com/olaure01/ollibs/blob/master/README.md"
+maintainer: "olivier.laurent@ens-lyon.fr"
+authors: [
+  "Olivier Laurent"
+]
+license: "LGPL-3.0-or-later"
+
+depends: [
+  "coq" { >= "8.12.0" & < "8.12.1" }
```

Comment:
```suggestion
  "coq" { >= "8.12.0" & < "8.13" }
```
The current version seems overly conservative. Coq devs do their best to _not_ break libraries with point releases.

Reply:
OK thanks, it was just a matter of being cautious.